packages/desktop-gui/src/runs/dashboard-banner.jsx

```diff
@@ -0,0 +1,40 @@
+import _ from 'lodash'
+import React, { Component } from 'react'
+
+export default class DashboardBanner extends Component {
+  render () {
+    return (
+      <div className='dashboard-banner'>
+        <div className='left-vector' />
+        <div className='right-vector' />
+        <div className='dashboard-banner-db'>
+          {_.map(['pass', 'fail', 'pass'], (test, i) => (
+            <div key={`test-${i}`} className={`dashboard-banner-test ${test}`}>
+              <div className='dashboard-banner-test-left'>
+                <div className='dashboard-banner-test-title'>
+                  <div className='test-title-top'>
+                    <i className='fas fa-check' />
+                    {_.capitalize(test)}ed test
+                  </div>
+                  <div className='test-title-bottom'>
+                    <div className='fake-text' style={{ width: '45px' }} />
```

Comment:
Maybe these widths should go in the css? Could use `nth-child()` for these ones in particular.

Reply:
Yeah...that’s what I did initially. Looked really messy and not as clear which is which.  But I guess inline styles are gross too.